cybergis/general/connection.py

```diff
@@ -60,7 +62,9 @@ def login(self, *args, **kwargs):
                 self._login_with_key()
             elif self.user_pw is None:
                 print("input password for {}".format(self.user_name))
-                self.user_pw = getpass()
+                self.user_pw = getpass.getpass()
```

Comment:
@CarnivalBug Since we haven't found a way to show a password textbox in notebook, we should commet out the two lines under this IF statement and "raise NotImplementedError()"